src/components/layout/be-square/style/nav.js

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components'
+import { Container, SectionContainer } from 'components/containers'
+import { LinkButton } from 'components/form'
+import device from 'themes/device'
+
+export const ButtonWrapper = styled(LinkButton)`
+    max-width: 102px;
```

Comment:
Should this be set to `min-width` and have `width` set to auto to cater for all the potential languages?